src/app/components/token-network-actions/token-network-actions.component.ts

```diff
@@ -7,14 +7,21 @@ import { Connection } from '../../models/connection';
     styleUrls: ['./token-network-actions.component.css']
 })
 export class TokenNetworkActionsComponent implements OnInit {
+    @Input() id: string;
     @Input() connection: Connection;
+    @Input() chainId: number;
```

Comment:
Since we are only using the `chainId` to do a boolean check wouldn't it be better to have a `boolean` property and hide the check, in the previous component instead of exposing it to the actions component?

Reply:
Yes that's better. Thanks!